inference/core/models/object_detection_base.py

```diff
@@ -156,7 +156,7 @@ def postprocess(
         max_detections: int = DEFAUlT_MAX_DETECTIONS,
         return_image_dims: bool = False,
```

Comment:
We don't use `return_image_dims` anymore, right? If so, let's remove all references to it.